Global/Eclipse.gitignore

```diff
@@ -49,3 +49,6 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Scala IDE specific (Scala & Java development for Eclipse)
```

Comment:
If this is already covered by `Scala.gitignore`, this seems to be redundant, right?